anki-editor.el

```diff
@@ -173,6 +173,12 @@ use Anki's builtin LaTeX support."
 are missing."
   :type '(repeat string))
 
+(defcustom anki-editor-field-alias nil
+  "Alist mapping note type to a list of cons.
+Each cons cell maps a field name alias to the corresponding to
+model field name."
```

Comment:
I think this is worded a bit awkwardly; what does "list of cons" mean in this context? Perhaps giving an example of `(OLD . NEW)` pairs would clear this up a bit

Reply:
Thanks for the comment. Will update with an example.